tests/__init__.py

```diff
@@ -35,3 +35,18 @@ def get_testdata(*paths):
     path = os.path.join(tests_datadir, *paths)
     with open(path, 'rb') as f:
         return f.read()
+
+
+# FIXME: delete after dropping py2 support
+# Monkey patch the unittest module to prevent the
+# DeprecationWarning about assertRaisesRegexp -> assertRaisesRegex
```

Comment:
can we solve it without a monkey patch?

Reply:
Right, according to #3846 it's "planned for Q4 2019 or Q1 2020" (:tada:) given that this is only used in the tests it should be fine IMHO